src/main/java/LineTokenizer.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class LineTokenizer {
+    Map<String, ArrayList<Integer>> newMapWords;
+    List<Integer> linesWithWord;
```

Comment:
Why these fields have default access modifier?

Reply:
Do you need the `linesWithWord` variable?

Reply:
Can you rename `newMapWords`?

Reply:
newMapWords - renamed + added private modifier;
linesWithWord - deleted